src/node_kernels/SDRSSTAMSNodeKernel.C

```diff
@@ -59,9 +59,18 @@ SDRSSTAMSNodeKernel::setup(Realm& realm)
   sigmaWTwo_ = realm.get_turb_model_constant(TM_sigmaWTwo);
   betaOne_ = realm.get_turb_model_constant(TM_betaOne);
   betaTwo_ = realm.get_turb_model_constant(TM_betaTwo);
-  gammaOne_ = realm.get_turb_model_constant(TM_gammaOne);
-  gammaTwo_ = realm.get_turb_model_constant(TM_gammaTwo);
   tkeProdLimitRatio_ = realm.get_turb_model_constant(TM_tkeProdLimitRatio);
+
+  lengthScaleLimiter_ = realm.solutionOptions_->lengthScaleLimiter_;
+  if (lengthScaleLimiter_) {
+    const NodeKernelTraits::DblType earthAngularVelocity = realm.solutionOptions_->earthAngularVelocity_;
+    const NodeKernelTraits::DblType pi = std::acos(-1.0);
```

Comment:
you can also use `M_PI` which is hardcoded in `math.h` to replace creating Pi variables.  Not requesting a change. Merely an FYI.

Reply:
Thanks.